### ISSUE TITLE
404 Not Found error when trying to download from texstudio.org

### ISSUE THREAD

**Issue by wujastyk** [texstudio-org/texstudio#2217]
Firefox 99.

### Expected behavior
Fetch a deb file from
![image](https://user-images.githubusercontent.com/762246/161989949-f5168edd-b014-48bc-a8c9-e3ceceaad708.png)

i.e.,

https://provo-mirror.opensuse.org/repositories/home:/jsundermeyer/xUbuntu_20.04/amd64/texstudio_4.2.2-1+3.1_amd64.deb

### Actual behavior
![image](https://user-images.githubusercontent.com/762246/161989601-534463c6-4b20-4861-9992-030796b04746.png)

**Comment by wujastyk** [comment]
![image](https://user-images.githubusercontent.com/762246/162010440-2d4e5b91-41ce-45d5-ac31-4aecc98a965b.png)

FF 99 on Linux Mint (this is a deb, after all :-)

Incidentally, this issue is only a week or two old.  I've been downloading TeXStudio from the website for many years without a hitch.

**Comment by sunderme** [comment]
we don‘t habe control over suse build service and its mirrors.

**Comment by sunderme** [comment]
the links (download.opensuse.org...) works fine here.

**Comment by sunderme** [comment]
by the way, on ubuntu, I would recommend the ppa, see webpage.

**Comment by wujastyk** [comment]
Yes, I've had the ppa in my Ubuntu software update system for years.  It's the failure of that that first alerted me to the download problem.  The ppa didn't work, so I went to the website and that didn't work either.  Then I went to `http://download.opensuse.org/repositories/home:/jsundermeyer/xUbuntu_18.04/amd64/` and that worked fine.  So I was able to upgrade.

As I write this, the ppa is working correctly again.  But the texstudio.org website is still giving me the [bad redirect](http://provo-mirror.opensuse.org/repositories/home:/jsundermeyer/xUbuntu_20.04/amd64/texstudio_4.2.2-1+3.1_amd64.deb) and a 404.
